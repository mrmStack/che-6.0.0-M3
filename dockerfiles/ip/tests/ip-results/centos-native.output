1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9001 qdisc pfifo_fast state UP qlen 1000
    link/ether 06:81:cd:14:be:b0 brd ff:ff:ff:ff:ff:ff
    inet 172.30.21.70/24 brd 172.30.21.255 scope global dynamic eth0
       valid_lft 2583sec preferred_lft 2583sec
    inet6 fe80::481:cdff:fe14:beb0/64 scope link
       valid_lft forever preferred_lft forever
3: br-64241850e9cf: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN
    link/ether 02:42:96:ae:8b:14 brd ff:ff:ff:ff:ff:ff
    inet 172.18.0.1/16 scope global br-64241850e9cf
       valid_lft forever preferred_lft forever
    inet6 fe80::42:96ff:feae:8b14/64 scope link
       valid_lft forever preferred_lft forever
4: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN
    link/ether 02:42:f7:a5:e1:80 brd ff:ff:ff:ff:ff:ff
    inet 172.17.42.1/16 scope global docker0
       valid_lft forever preferred_lft forever
    inet6 fe80::42:f7ff:fea5:e180/64 scope link
       valid_lft forever preferred_lft forever
5785: vethd38001c@if5784: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether ce:96:ab:87:85:51 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::cc96:abff:fe87:8551/64 scope link
       valid_lft forever preferred_lft forever
5787: vetha4cd0c7@if5786: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether aa:8e:d0:82:49:d3 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::a88e:d0ff:fe82:49d3/64 scope link
       valid_lft forever preferred_lft forever
5789: veth784f0a5@if5788: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether f6:23:3d:0d:11:22 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::f423:3dff:fe0d:1122/64 scope link
       valid_lft forever preferred_lft forever
5791: vethb7e8eb7@if5790: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether 7a:af:c0:89:67:da brd ff:ff:ff:ff:ff:ff
    inet6 fe80::78af:c0ff:fe89:67da/64 scope link
       valid_lft forever preferred_lft forever
5793: veth4afdbe2@if5792: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether ca:3f:1d:ba:e2:c6 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::c83f:1dff:feba:e2c6/64 scope link
       valid_lft forever preferred_lft forever
5795: vethc5a890f@if5794: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether 9e:b1:f3:ff:b8:55 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::9cb1:f3ff:feff:b855/64 scope link
       valid_lft forever preferred_lft forever
5797: vethf6060e0@if5796: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether f6:e3:af:71:9f:00 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::f4e3:afff:fe71:9f00/64 scope link
       valid_lft forever preferred_lft forever
5799: vethe9f72b5@if5798: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether 92:e5:70:94:eb:d9 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::90e5:70ff:fe94:ebd9/64 scope link
       valid_lft forever preferred_lft forever
5801: vethb2884a1@if5800: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether 8e:c5:1c:9a:be:11 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::8cc5:1cff:fe9a:be11/64 scope link
       valid_lft forever preferred_lft forever
3753: br-4770a54d50f0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP
    link/ether 02:42:1c:a4:35:cd brd ff:ff:ff:ff:ff:ff
    inet 172.19.0.1/16 scope global br-4770a54d50f0
       valid_lft forever preferred_lft forever
    inet6 fe80::42:1cff:fea4:35cd/64 scope link
       valid_lft forever preferred_lft forever
5803: veth6de36e1@if5802: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue master br-4770a54d50f0 state UP
    link/ether aa:e7:35:dc:70:28 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::a8e7:35ff:fedc:7028/64 scope link
