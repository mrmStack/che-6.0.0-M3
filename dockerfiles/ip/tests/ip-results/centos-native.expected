172.30.21.70
